### PR TITLE
Add direction to stores feature in agent application

### PR DESCRIPTION
**Pull request by vahinim**
Add list_of_stores_with_direction directory contains the files of this application.
Update other files to use this feature.

### CODE REVIEW

File: frontend/gpay_agent/lib/agent_datamodels/store.dart [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    var response = await http.post(
+        "https://fos-tracker-278709.an.r.appspot.com/store/phone",
+        body: jsonEncode(<String, String>{"storePhone": phone}));
+    if (response.statusCode == 200) {
+      print('found number');
+      try {
+        LineSplitter lineSplitter = new LineSplitter();
+        List<String> lines = lineSplitter.convert(response.body);
+        String jsonString = lines[0];
+        print(jsonString);
```
Comment (krati-tiwari) [inline frontend/gpay_agent/lib/agent_datamodels/store.dart]: remove

File: frontend/gpay_agent/lib/agent_datamodels/store.dart [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        creationDateTime: json['creationDateTime']
+    );
+  }
+
+  /// Fetches store from the spanner database.
+  static Future<Null> fetchStore(String phone) async {
+    print('started' + phone);
+    var response = await http.post(
+        "https://fos-tracker-278709.an.r.appspot.com/store/phone",
+        body: jsonEncode(<String, String>{"storePhone": phone}));
```
Comment (krati-tiwari) [inline frontend/gpay_agent/lib/agent_datamodels/store.dart]: add space after String>

**Review by krati-tiwari — APPROVED** [review]

File: frontend/gpay_agent/lib/agent_datamodels/store.dart (hunk 1 above)
Comment (vahinim) [reply frontend/gpay_agent/lib/agent_datamodels/store.dart]: removed

File: frontend/gpay_agent/lib/agent_datamodels/store.dart (hunk 2 above)
Comment (vahinim) [reply frontend/gpay_agent/lib/agent_datamodels/store.dart]: added

**Pull request merged**